### ISSUE TITLE
Polymer growing silent on the public front - will resources taken away be replaced? 

### ISSUE THREAD

**Issue by dman777** [notwaldorf/ama#79]
For the past 3 years, I have always tried to push Polymer into my projects of my employment. I convinced a team in my last job to use Polymer in their next project AND I just started a new job where they are using Polymer on their existing product! After 3 years of being in the market,  I see Polymer is finally starting to take off in the market! 

But at the same time, Polymer has fallen silent. I get no replies from the Polymer team on twitter. Rob Dobson has stopped doing Polycasts. Tyler Rhodes hasn't replied to my email. And worse, they have taken Eric Bidelman off the project.... the public front man of Polymer. 

Even though Polymer 3 is still going to be released, it still looks like Google is abandoning Polymer. They took away resources, there is no public front or any interaction with the community. Other engineers on my team for the Polymer project have the same impression. What a time to go dim when Polymer is finally taking off in the market! 

Will the resources that were taken away - Eric Bidelman, Rob Dobson's Podcasts, Tyler Rhodes, be replaced? Will the Polymer team or a new front man get involved with the community like Eric Bidelman did so well? 

Thank you,
-Darin

**Comment by notwaldorf** [comment]
Hiya!

I can promise you Google is not abandoning Polymer, and Polymer is as strong and busy as ever. As we're finishing up Polymer 3, the team is super busy already working on the next steps: pwa templates, a small and fast base element based on `lit-html`, shady css polyfill improvements, template instantiation, spec work for shadow DOM theming, faster elements, etc etc etc. Polymer continues to be used by very large teams and projects (such as YouTube, EA, etc), so it's very unlikely it will ever get dropped. :)

Now, to answer some of your specific questions:
- I have never heard of Tyler Rhodes, and he has never been associated with the Polymer team, so there's nothing much we can do about him not responding to your emails 🙂. That being said, emailing or pinging individual Polymer team members with "I have a problem in my application" isn't super productive -- these questions are best asked on the Slack channel, where there's more eyeballs looking at it, on the Stack Overflow, where they can be indexed and come up in search results for other people, or as an issue on the repo you think is causing you grief, with repro steps. 
- @ebidel was never a dedicated Polymer dev rel -- he works on Chrome, so he was dev relling Web Components as a new web technology specifically. The fact that he was also involved with Polymer was a side effect of the Polymer team being the one that started the work on Web Components (and not the other way around). He's not really been involved since before I joined the team, which was 3 years ago. Since then, he's been working on other projects, such as Lighthouse. Similarly, @robdodson doesn't really work on Polymer, but on Web Components as a whole. He's not focusing on one library (Polymer), and recently he's been working on getting other libraries to adopt the underlying technology. That's why he made custom-elements-everywhere.com. Pinging him on Twitter also doesn't work very well because his mentions are a deluge of tech support requests. This is why asking questions on Stack Overflow is better, since they're indexable and searchable, and can help more than one person at a time :)
- since Rob's focus has shifted, @e111077 has been recording a couple of Polycasts that should be released soon, but we saw this as an opportunity to take a step back and re-evaluate whether we _should_ continue doing Polycasts. They take a lot of time and effort to record (a couple of hours of work to plan out, and about an hour to record for a 5-10 minute segment), and they go out of date sooooo fast. Essentially all the Polycasts we've recorded up to this point are throw away because they deal with older Polymer syntax, and it's a very frustrating experience for users who google them on youtube, and get something with the right name and the wrong syntax. As a result, we've been trying to focus more on [blog posts](https://www.polymer-project.org/blog/) and keeping the [code labs](https://codelabs.developers.google.com/) up to date.
- now Twitter ... it is very true we are bad at Twitter. Since we're a bit of a strange team because we don't have a dedicated community manager or a dev rel, it means that the only people looking at the Twitter account are the engineers, and most of our team members are just not Twitter users. That, combined with the fact that Twitter is a terrible medium for tech support (280 characters at a time gives no context for your actual problem), means that it's not really a feasible way to get your problems sorted out. However, we are creating a community rotation, where we all take turns steering the Twitter boat, so hopefully that will improve in the future!
